# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/coreutils/cross.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./lib/putenv.c.orig Mon Apr 22 18:21:01 2002
+++ ./lib/putenv.c      Mon Apr 22 18:23:04 2002
@@ -31,8 +31,10 @@
    with a non-const argument.  That would conflict with the declaration of
    rpl_putenv below (due to the #define putenv rpl_putenv from config.h).  */

+#if 0
 char *malloc ();
 void free ();
+#endif

 #if defined (__GNU_LIBRARY__) || defined (HAVE_STRING_H)
 # include <string.h>
--- ./lib/getline.c.orig        Mon Apr 22 18:23:35 2002
+++ ./lib/getline.c     Mon Apr 22 18:24:06 2002
@@ -33,11 +33,13 @@

 #if defined __GNU_LIBRARY__ && HAVE_GETDELIM

+#if 0
 int
 getline (char **lineptr, size_t *n, FILE *stream)
 {
   return getdelim (lineptr, n, '\n', stream);
 }
+#endif

 #else /* ! have getdelim */

